lib/node_modules/@stdlib/math/base/special/factorial2/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env julia
```

Comment:
Did some investigating. Based on the SciPy and Boost implementations, the double factorial is simply not implemented for doubles. In the code above, if you cast `x` to integers, then the fixtures work.

As a result, unlikely to get either of those two reference implementations to provide test fixtures. 

Further, I am willing to hazard a guess that the Gamma relation is not straightforward. Boost only applies a Gamma approximation for odd integers. SciPy applies two different approximations based on whether an input integer is odd or even. While the double factorial __can__ be extended to reals, neither of those two implementations can serve as reference implementations.